doc/rst/technotes/gpu.rst

```diff
@@ -99,3 +95,27 @@ kernels for the ``forall`` loops in the function.
   }
 
 
+Multi-Locale Support
+--------------------
+
+As of Chapel 1.27.0 the GPU locale model may be used alongside communication
+layers (values of ``CHPL_COMM``) other than ``none``. This enables programs to
+use of GPUs across nodes.
+
+In this mode, normal inter-node PGAS style access is supported outside of loops
```

Comment:
Use "remote access" instead of "inter-node PGAS style" access?

Reply:
Sure, updated